### ISSUE TITLE
DVC studio not able to pic project which is not in root dir of DVC Studio 

### ISSUE THREAD

**Issue by RajeshThakur1** [iterative/studio-support#55]
Hi DVC Team,

we were exploring the DVC and we found one bug

step to replicate

step1:-
I have my project in github which has the root dir is the root
inside of root dir, we have many projects, one project is related to DVC

but when we are trying to configure DVC studio with our child dir

![image](https://user-images.githubusercontent.com/46486711/180001050-b8575d1d-50de-479e-8622-9c8c68c5e26f.png)

but somehow after configuring the project, DVC is picking the URL like <remote_repo><child_dir>
actually, it should pick <remote_repo>/<child_dir>

**Comment by shcheklein** [comment]
@RajeshThakur1 thanks for the report. Could you clarify please:

`DVC is picking the URL like <remote_repo><child_dir>` 

Where do you see this URL?
Are still able to see the data from the `training_and_deployment_pipeline`?

**Comment by alex000kim** [comment]
@RajeshThakur1 could please post the project tree here? (whatever you are comfortable sharing publically)

**Comment by Suor** [comment]
I looked into this and it's in a weird state: a real subrepo with `.dvc` dir is managed like a slice. My guess is that there were no `.dvc` in the subdir the first time repo was parsed and Studio failed to recognize that a repo with a subdir is no longer a slice but a real dvc repo once that dir was added - this is a bug in Studio.

Unfortunately simply deleting and recreating the view won't fix this. However, deleting a repo itself will do. 

So I will manually fix this for now and issue a reparse.
